numba/tests/test_withlifting.py

```diff
@@ -800,6 +803,34 @@ def f():
 
         self.assertEqual(f(), 1 + 3)
 
+    def test_objmode_gv_name(self):
+        @njit
+        def global_name():
+            with objmode(val='gv_type'):
```

Comment:
Shouldn't this be the following?
```suggestion
            with objmode(val=gv_type):
```

Reply:
Testing the use of global variable is done in `test_objmode_gv_variable`.
But now i wonder if we actually need the support for spelling type as string at all.

Reply:
Removed the support of using `str` to reference global variables.